variables.tf

```diff
@@ -83,10 +97,10 @@ variable "tagName" {
   description = "Name tag for the servers"
 }
 
-variable "vpc_id" {
-  description = "The AWS VPC ID which you want to deploy your instances"
+variable "user_data" {
+  default = ""
 }
 
-variable "associate_public_ip_address" {
-  default = false
+variable "vpc_id" {
```

Comment:
in the future i would like us to accept human-readable VPC identifiers (using the `aws_vpc` data source and filter expressions, like i was trying to do yesterday) instead of VPC IDs, but that's a future feature request, not a reason to block this change